src/main/java/mate/academy/springboot/datajpa/service/mapper/RequestDtoMapper.java

```diff
@@ -1,5 +1,5 @@
 package mate.academy.springboot.datajpa.service.mapper;
 
 public interface RequestDtoMapper<D, T> {
-    T mapToModel(D dto);
+    T mapToModel(D requestDto);
```

Comment:
```suggestion
    T mapToModel(D dto);
```
In my opinion, it looks better)